app/assets/onepager/css/main.css

```diff
@@ -560,6 +560,18 @@ body #main a:not(.button):hover {
   margin: auto;
 }
 
+.alert {
+  position: absolute;
+}
+#footer {
+  position: absolute;
```

Comment:
you could add `position-absolute` as opposed to redefining the style

Reply:
^ this still hasn't been addressed

Reply:
Fixed